tsl/test/expected/cagg_joins-12.out

```diff
@@ -188,21 +322,8 @@ NOTICE:  refreshing continuous aggregate "cagg_on_cagg"
 HINT:  Use WITH NO DATA if you do not want to refresh the continuous aggregate on creation.
 DROP MATERIALIZED VIEW cagg_on_cagg CASCADE;
 NOTICE:  drop cascades to 2 other objects
-DETAIL:  drop cascades to table _timescaledb_internal._hyper_6_35_chunk
-drop cascades to table _timescaledb_internal._hyper_6_36_chunk
---Error out when real time aggregation is enabled
-CREATE MATERIALIZED VIEW conditions_summary_daily_realtime
-WITH (timescaledb.continuous, timescaledb.materialized_only = FALSE) AS
-SELECT time_bucket(INTERVAL '1 day', day) AS bucket,
-   AVG(temperature),
-   MAX(temperature),
-   MIN(temperature),
-   name
-FROM conditions, devices
-WHERE conditions.device_id = devices.device_id
-GROUP BY name, bucket;
-ERROR:  real-time continuous aggregates are not supported with joins
-HINT:  set materialized_only to true
+DETAIL:  drop cascades to table _timescaledb_internal._hyper_8_41_chunk
+drop cascades to table _timescaledb_internal._hyper_8_42_chunk
```

Comment:
Would suggest to not print the detailed message here, so set verbosity to `terse`.